### DISCUSSION TITLE
Welcome to domino-rest Discussions!

### DISCUSSION THREAD

**Discussion by vegegoku** [DominoKit/domino-rest#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by natros** [comment]
Hi!,

I'm here to say thank you for releasing domino-rest to maven central. 
Now, I cant start migrating from restygwt.

Thank you.

**Reply by FrankHossfeld** [reply]
domino-rest is in Maven Central? Did I miss something?? Great news!

**Reply by vegegoku** [reply]
> Hi!,
> 
> I'm here to say thank you for releasing domino-rest to maven central.
> Now, I cant start migrating from restygwt.
> 
> Thank you.

We appreciate any kind of test and feedback, we still didnt update the wiki and there is some breaking changes in the beta release and we changed the some artifacts ids

**Reply by vegegoku** [reply]
> domino-rest is in Maven Central? Did I miss something?? Great news!

Yes there is a beta but with some breaking changes..nothing big ..maven dependency artifact id and package names

**Reply by FrankHossfeld** [reply]
I tried, but failed ... think to the changes. I was not able to find org.dominokit:domino-rest-apt:jar:1.0.0-beta-2. Will wait until the doc I updated. (Or, is the doc already updated?)

**Reply by natros** [reply]
> I tried, but failed ... think to the changes. I was not able to find org.dominokit:domino-rest-apt:jar:1.0.0-beta-2. Will wait until the doc I updated. (Or, is the doc already updated?)

now it's `org.dominokit:domino-rest-processor:1.0.0-beta-2`

https://search.maven.org/search?q=g:org.dominokit%20AND%20v:1.0.0-beta-2

**Reply by vegegoku** [reply]
We are trying to unify how we name our artifacts and packages across different projects

**Reply by vegegoku** [reply]
I have updated the wiki to reflect the recent changes but this is not final and we are still investigating new ways to handle development snapshots.
